tests/testthat/test-derive_param_tte.R

```diff
@@ -974,3 +974,129 @@ test_that("list_tte_source_objects Test 15: expected objects produced", {
 
   expect_dfs_equal(expected_output, observed_output, keys = c("object"))
 })
+
+## Test 15: derive_param_tte detects duplicates when check_type = 'warning' ----
```

Comment:
Please move the new tests before "Test 14". ("Test 14" and "Test 15" test `list_tte_source_objects()` and not `derive_param_tte()`). Then all tests for `derive_param_tte()` are in the same section.